test/api/v3/integration/tasks/PUT-tasks_id.test.js

```diff
@@ -176,6 +177,42 @@ describe('PUT /tasks/:id', () => {
       expect(savedDaily.reminders[0].id).to.equal(id1);
       expect(savedDaily.reminders[1].id).to.equal(id2);
     });
+
+    it('can set a shortName if the shortName does not already exist', async () => {
+      let savedDaily = await user.put(`/tasks/${daily._id}`, {
+        shortName: 'short-name',
+      });
+
+      expect(savedDaily.shortName).to.eql('short-name');
+    });
+
+    it('ignores shortName if it is already set', async () => {
```

Comment:
That said, if we allowed short names to be edited up front and then changed our minds and decided that they can't be edited, that would be a breaking change. But deciding that they can't be edited and then allowing them to be edited later, that would be a new feature and not a breaking change. 

So if you're still not convinced, I'll just carry on with the version of this feature where you can only set it once. (And hope that eventually you come around 😛  ) 

Ultimately, this is intended for use by very simple integrations that require id + api-token + short-name, so I want to make this as easy for users as possible.